frontend/src/components/table/TableItem.js

```diff
@@ -195,9 +203,9 @@ class TableItem extends PureComponent {
    * @param {object} [item] - widget data object
    * @param {boolean} [select] - flag if selected cell should be cleared
    */
-  handleEditProperty = (e, property, focus, item, select) => {
+  handleEditProperty = (e, property, focus, item, select, mark) => {
```

Comment:
Same -> add a comment what `mark` property is.

Reply:
Done